pages.hi/common/broot.md

```diff
@@ -0,0 +1,28 @@
+# broot
+
+> निर्देशिकाओं को पथ-प्रदर्शित करने का एक बेहतर तरीका।
+> अधिक जानकारी: <https://github.com/Canop/broot>।
+
+- `br` shell कार्य को स्थापित या पुनर्स्थापित करें:
+
+`broot --install`
+
+- वर्तमान निर्देशिकाएं पथ-प्रदर्शित करें:
```

Comment:
Checked this with Google translate shouldn't it be `Navigate the current directory:`

Reply:
@adityarjadhav could you please check this?